pkg/kubelet/kubelet.go

```diff
@@ -2407,6 +2413,42 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		}
 		oldNodeUnschedulable = node.Spec.Unschedulable
 	}
+
+	var newNodeAcceptingCondition api.NodeCondition
+	if kl.accepting {
+		newNodeAcceptingCondition = api.NodeCondition{
+			Type:              api.NodeAccepting,
+			Status:            api.ConditionTrue,
+			Reason:            "KubeletAccepting",
+			Message:           "kubelet is accepting work",
+			LastHeartbeatTime: currentTime,
+		}
+	} else {
+		newNodeAcceptingCondition = api.NodeCondition{
+			Type:              api.NodeAccepting,
+			Status:            api.ConditionFalse,
+			Reason:            "KubeletSwitchedOff",
+			Message:           "an operator disabled this kubelet",
```

Comment:
Are these messages visible to users? It seems like it might make sense to start this message with a capital letter.